rules-reviewed/eap7/eap6/eap6-xml.windup.xml

```diff
@@ -168,6 +168,199 @@
                 </iteration>
             </perform>
         </rule>
+        <rule id="eap6-xml-08000">
+            <when>
+                <xmlfile matches="/*[local-name()='jboss-web']/*[local-name()='replication-config']/*[local-name()='replication-trigger']" in="jboss-web.xml"/>
+            </when>
+            <perform>
+                <hint title="Web Session Clustering config changes in EAP 7" effort="1" severity="optional">
```

Comment:
All the remaining rules for `jboss-web.xml` clustering changes have a message of "Web Session Clustering config XXX changes in EAP 7", only this one has "Web Session Clustering config changes in EAP 7".

Either all should have "Web Session Clustering config changes in EAP 7", or this one should be changed to "Web Session Clustering config replication-config changes in EAP 7".